demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -76,7 +76,7 @@ def __init__(self, input: str, output: str = '', file_type: str = '', no_demisto
     def get_output_path(self):
         """Get processed output path
         """
-        output_path = self.output
+        output_path = Path(self.output).resolve()
```

Comment:
Maybe better to add this in the ctor with both input and output?

Reply:
true, will do, thanks